# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/bsd-games/pc_sym_hack.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./hack/hack.termcap.c.orig	2005-06-01 13:08:46.361608016 +0200
+++ ./hack/hack.termcap.c	2005-06-01 13:09:16.788982352 +0200
@@ -80,7 +80,7 @@
 char           *HO, *CL, *CE, *UP, *CM, *ND, *XD, *BC, *SO, *SE, *TI, *TE;
 static char    *VS, *VE;
 static int      SG;
-char            PC = '\0';
+char            PC_my = '\0';
 char           *CD;		/* tested in pri.c: docorner() */
 int             CO, LI;		/* used in pri.c and whatis.c */
 
@@ -101,7 +101,7 @@
 	if (tgetent(tptr, term) < 1)
 		error("Unknown terminal type: %s.", term);
 	if ((pc = tgetstr("pc", &tbufptr)) != NULL)
-		PC = *pc;
+		PC_my = *pc;
 	if (!(BC = tgetstr("bc", &tbufptr))) {
 		if (!tgetflag("bs"))
 			error("Terminal must backspace.");
